waddrmgr/db.go

```diff
@@ -623,39 +711,49 @@ func deserializeImportedAddress(addressID []byte, row *dbAddressRow) (*dbImporte
 		return nil, managerError(ErrDatabase, str, nil)
 	}
 
-	retRow := dbImportedAddressRow{
+	retRow := dbImportedPubKeyAddressRow{
 		dbAddressRow: *row,
 	}
 
-	pubLen := binary.LittleEndian.Uint32(row.rawData[0:4])
+	var offset uint32
```

Comment:
I'd avoid the use of the offset variable here since all of the offsets are constant (the pubLen below can be removed, see above comment).  No need to use a var to keep track of exactly where the next byte needs to be read from.